src/Components/Contacts/Contacts.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './contacts.css';
+
+const Contacts = (props) => {
+    return (
+        <div className={"Person"}>
```

Comment:
This should only be a string unless you're trying to use jsx to change its value in some way